packages/at_client/lib/src/preference/at_client_preference.dart

```diff
@@ -43,7 +43,7 @@ class AtClientPreference {
 
   /// Maximum data size a secondary can store. Temporary solution. Have to fetch this from
```

Comment:
we have to change this documentation. See # 3 in the ticket.

Reply:
Updated the dart docs